### PR TITLE
fix(ci) : upgrade node and golang versions

### PR DESCRIPTION
**Pull request by camilamacedo86**
## Motivation
- Issue faced when the CI is building UI (integrity checksum failed when using sha1:)

## What
- Fix `integrity checksum failed when using sha1:` issue
- Upgrade node and go versions in the CI

### CODE REVIEW

File: ui/package.json [hunk 1]
```diff
@@ -9,7 +9,7 @@
     "@patternfly/react-core": "2.3.6",
     "@patternfly/react-table": "1.0.7",
     "dotenv": "^6.2.0",
-    "lodash": "4.17.11",
+    "lodash": "^4.17.15",
```
Comment (craicoverflow) [inline ui/package.json]:
```suggestion
"lodash": "4.17.15",
```

File: ui/package.json [hunk 2]
```diff
@@ -9,7 +9,7 @@
...
     "lodash-es": "4.17.11",
```
Comment (craicoverflow) [inline ui/package.json]: Should this also be increased to 4.7.15 also?

File: .circleci/config.yml [hunk 3]
```diff
@@ -35,7 +35,7 @@ jobs:
     working_directory: ~/mobile-security-service
 
     docker:
-      - image: circleci/node:8.15.0
+      - image: circleci/node:12
```
Comment (craicoverflow) [inline .circleci/config.yml]: I believe we should set the Node version to 10. See that the Dockerfile also uses Node v10:

https://github.com/aerogear/mobile-security-service/blob/c29f31e90d667da1f01f1664aa15bf1dc5ef67f8/Dockerfile#L8

File: ui/package.json (hunk 1 above)
Comment (camilamacedo86) [reply ui/package.json]: It will not work and will cause the issue.

File: ui/package.json (hunk 2 above)
Comment (camilamacedo86) [reply ui/package.json]: I do not think so.

File: .circleci/config.yml (hunk 3 above)
Comment (camilamacedo86) [reply .circleci/config.yml]: 👍 doing

**Pull request merged**